README.md

```diff
@@ -19,25 +19,50 @@ From your shell or command line:
 
 `git clone https://github.com/microsoftgraph/ConsoleApp-DeltaQuery-DotNet`
 
-### Step 2:  Running this application with your Azure Active Directory tenant
+### Step 2:  Configuring your Azure AD tenant
 
-#### Register Sample app for your own tenant
+As a first step you'll need to:
 
-Using MSAL
-1.Before you can get a token from Azure AD v2.0 or Azure AD B2C, you'll need to register an application(https://apps.dev.microsoft.com/). For Azure AD v2.0, use the app registration portal. For Azure AD B2C, checkout how to register your app with B2C.
-2. Find the Application ID value and copy it to the clipboard.
-3. Configure Permissions for your application - select 'Read Mail'.
-4. Set a platform by clicking Add Platform, select Native.
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account or a personal Microsoft account.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant
+   (using **Switch Directory**).
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview)**.
 
+#### Register the client app (ConsoleApp-DeltaQuery-DotNet)
 
-###  Step 3: Setup sample project
+1. In **App registrations (Preview)** page, select **Register an Application**.
+1. When the **Register an application page** appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `ConsoleApp-DeltaQuery-DotNet`.
+   - In the **Supported account types** section, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
+   - Select **Register** to create the application.
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. In the list of pages for the app, select **Manifest**, and:
+   - In the manifest editor, set the ``allowPublicClient`` property to **true**
+   - Select **Save** in the bar above the manifest editor.
+1. In the list of pages for the app, select **Authentication**
+   - In the *Suggested Redirect URIs for public clients(mobile,desktop)*, check all the boxes so that the app can work with the MSAL libs used in the application.
```

Comment:
No: you should only check the urn:obo: …, etc.. !

:)

Reply:
unless you reuse the client ID ?

Reply:
Well spotted. 👍  
Should be resolved with the latest push.